components/admin_console/admin_definition.jsx

```diff
@@ -3428,6 +3452,7 @@ const AdminDefinition = {
                         type: Constants.SettingsTypes.TYPE_CUSTOM,
                         component: LDAPFeatureDiscovery,
                         key: 'LDAPFeatureDiscovery',
+                        isDisabled: it.not(it.userHasWritePermissionOnResource(RESOURCE_KEYS.AUTHENTICATION.LDAP)),
```

Comment:
This allows a user to request a trial license. Therefore, the user requires "manage_license_information" permission which is tied to ABOUT.EDITION_AND_LICENSE. 
The ABOUT PRs may need merged first.

```suggestion
                        isDisabled: it.not(it.userHasWritePermissionOnResource(RESOURCE_KEYS.ABOUT.EDITION_AND_LICENSE)),
```